.buildkite/pipeline.yml

```diff
@@ -24,8 +24,10 @@ steps:
 
   - label: "🧪 Tests"
     key: tests
+    plugins: [$CI_TOOLKIT]
     command: |
-      xcodebuild test -project GravatarApp.xcodeproj -scheme GravatarApp -destination 'platform=iOS Simulator,name=iPhone 16,arch=arm64'
+      install_swiftpm_dependencies
+      make test
```

Comment:
The move from calling `xcodebuild` here to a `make` task is so that if a dev wants to run the same command as CI does locally, they can do so.